test/function_call_test.js

```diff
@@ -82,19 +82,32 @@ describe('function calls', () => {
     `);
   });
 
-  it.skip('places parentheses in calls with multi-line function arguments after the closing brace', () => {
+  it('places parentheses in calls with multi-line function arguments after the closing brace', () => {
     check(`
       promise.then ->
+        a()
         b # c
       d
     `, `
       promise.then(function() {
+        a();
         return b; // c
       });
       d;
     `);
   });
 
+  it('places parentheses in calls with fat arrow function arguments after the closing brace', () => {
```

Comment:
Well I noticed that it did not transpile as what you think.
```coffee
promise.then -> 
  b # c		
d
```
into 
```js
promise.then(function(){
  return b ; // c;
})
d
```
Closing brace seems to worked well.
But Single-line and comments lead to different codes ( => , {} or not  .....) . 
Thats why I add 2 new test cases for them .
btw. I am not good at English , can you give me a hand rewriting descriptions?